src/pages/settings/ProfilePage.js

```diff
@@ -1,24 +1,236 @@
+                                        value={this.state.pronouns}
+                                        Icon={() => <Icon src={DownArrow} />}
+                                    />
+                                </View>
+                                {this.state.pronouns === pronounsMap.selfSelect ? (
+                                    <TextInput
+                                        style={styles.textInput}
+                                        value={this.state.selfSelectedPronouns}
+                                        onChangeText={selfSelectedPronouns => this.setState({selfSelectedPronouns})}
+                                        placeholder="Self-select your pronoun"
+                                    />
+                                ) : null}
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={styles.mb1}>
+                                    {Str.isSMSLogin(this.props.myPersonalDetails.login)
+                                        ? 'Phone Number' : 'Email Address'}
+                                </Text>
+                                <TextInput
+                                    style={[styles.textInput, styles.disabledTextInput]}
+                                    value={this.props.myPersonalDetails.login}
+                                    editable={false}
+                                />
+                            </View>
+                            <View style={styles.mb2}>
+                                <Text style={styles.mb1}>Timezone</Text>
+                                <RNPickerSelect
+                                    onValueChange={selectedTimezone => this.setState({selectedTimezone})}
+                                    items={timezones}
+                                    style={this.state.isAutomaticTimezone ? {
+                                        ...styles.picker,
+                                        inputIOS: [styles.picker.inputIOS, styles.textInput, styles.disabledTextInput],
+                                        inputAndroid: [
+                                            styles.picker.inputAndroid, styles.textInput, styles.disabledTextInput,
+                                        ],
+                                        inputWeb: [styles.picker.inputWeb, styles.textInput, styles.disabledTextInput],
+                                    } : styles.picker}
+                                    useNativeAndroidPickerStyle={false}
+                                    value={this.state.selectedTimezone}
+                                    Icon={() => <Icon src={DownArrow} />}
+                                    disabled={this.state.isAutomaticTimezone}
+                                />
+                            </View>
+                            <Checkbox
+                                label="Set my timezone automatically"
```

Comment:
This appears to render in the incorrect color/font:
![image](https://user-images.githubusercontent.com/2319350/111128120-d9b2b400-8574-11eb-8ab5-b51185e561a1.png)


Can you make sure it is using basically all of the same styles as `styles.textP`?